### ISSUE TITLE
Update linter.sh to properly test for jq [phab:D992]

### ISSUE THREAD

**Issue by kythe** [google/kythe#1325]
_Created by **michel-slm** at 2016-07-29 01:41:25_: 

The script previously attempts to use one invocation of `command` to test for both `shellcheck` and `jq`; while this would guard against shellcheck not being present, it would allow the block to be executed even if `jq` is not present.

Use two separate `command` invocations instead, and only execute the block if both commands succeed.

**Comment by kythe** [comment]
_**michel-slm** wrote at 2016-07-29 02:12:19_:

Trigger unit tests

**Comment by kythe** [comment]
_**schroederc** wrote at 2016-07-29 16:21:18_:

Looks good.  Please sign a CLA so we can get this landed: https://github.com/google/kythe/blob/master/CONTRIBUTING.md

**Comment by kythe** [comment]
_**schroederc** required changes at 2016-07-29 16:21:24_

**Comment by kythe** [comment]
_**michel-slm** wrote at 2016-07-29 17:01:47_:

I have signed the CLA (under the email address michael.silvanusgmail.com, but my GitHub username `michel-slm` is listed in that CLA signature).

To be sure, I've just signed the CLA again with another Google account that has the email address I added to CONTRIBUTORS.md, `michelmichel-slm.name` as an alternate email address. Also listing the GitHub username `michel-slm`.

Is this satisfactory? The Google CLA page says:
```
Ensure you submitted your contribution using one of the email addresses associated with the account you're signed in to and/or using the GitHub username you provided us with when you signed the CLA. This is how we match your submission with your CLA.
```

**Comment by kythe** [comment]
_**schroederc** wrote at 2016-07-29 17:16:37_:

Sounds good!

**Comment by kythe** [comment]
_**schroederc** accepted this revision at 2016-07-29 17:16:37_

**Comment by kythe** [comment]
_**schroederc** closed via commit cb45cd9f391e3f50344acf6c1df33f61031b99e4 at 2016-07-29 17:18:43_
